include/quic/btstream.hpp

```diff
@@ -10,7 +10,7 @@ namespace oxen::quic
     using time_point = std::chrono::steady_clock::time_point;
 
     // timeout is used for sent requests awaiting responses
-    inline constexpr std::chrono::seconds TIMEOUT{10};
+    inline constexpr std::chrono::milliseconds TIMEOUT{10s};
```

Comment:
I think a lower default of like 2s would be better here, and if a message wants a longer timeout it can specify.

Reply:
also, since this is constexpr, it should probably be named something like `DEFAULT_TIMEOUT`